microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -361,6 +361,7 @@ def enable_uio_hv_generic_for_nic(node: Node, nic: NicInfo) -> None:
 def initialize_node_resources(
     node: Node, log: Logger, variables: Dict[str, Any], pmd: str
 ) -> DpdkTestResources:
+    dpdk_source = variables.get("dpdk_source", "")
```

Comment:
It can like below to apply default value:

```python
dpdk_source = variables.get("dpdk_source", _DPDK_STABLE_GIT)
```

Reply:
It's also good to print out what value is got here. So if there is typo on variable names, it would be found easier.